config/application.rb

```diff
@@ -343,6 +344,30 @@ def init_dynflow
     def setup_auditing
       Audit.send(:include, AuditSearch)
     end
+
+    def notify_deprecations
```

Comment:
wouldn't it be better to have a sepparate `config/initializers/<i.e. depracate_mysql.rb>` file?

Reply:
Yes, we actually have an existing `deprecations.rb` initializer, so moving it there.